packages/grafana-data/src/types/featureToggles.gen.ts

```diff
@@ -39,4 +39,6 @@ export interface FeatureToggles {
   validatedQueries?: boolean;
   swaggerUi?: boolean;
   featureHighlights?: boolean;
+  liveDashboardDiscussions?: boolean;
```

Comment:
not sure this needs to be related to live specifically? That seems more like an implementation detail. The feature is just `comments`

Reply:
Fixed, + updated PR description